lib/firebase/notifications.dart

```diff
@@ -104,12 +105,29 @@ class Notifications {
 
   /// Retrieves and saves the FCM token for the device.
   Future<void> getFCMToken() async {
+    if (Platform.isIOS) {
+      await _firebaseMessaging.getAPNSToken();
+    }
+
     final fCMToken = await _firebaseMessaging.getToken();
     final user = await _authService.getUserData();
+
+    ZOLogger.logMessage('FCM token: $fCMToken', isError: true);
     if (user == null) {
       ZOLogger.logMessage("User is not logged in, nothing to update");
       return;
     }
     _entityService.saveFCMToken(user.entityId, fCMToken);
   }
+
+  void listenToTokenRefresh() async {
```

Comment:
Thanks for adding 🔝 

Super nit: I guess this `async` is not needed, as you just setting a listener

Reply:
Removed 👍